heliopy/data/stereo.py

```diff
@@ -84,11 +84,49 @@ def mag_l1_rtn(spacecraft, starttime, endtime):
 
 
 def magplasma_l2(spacecraft, starttime, endtime):
-    units = OrderedDict([('Q_FLAG', u.dimensionless_unscaled)])
     return _stereo(starttime, endtime, spacecraft, 'L2_MAGPLASMA_1M',
-                   units=units, intervals='monthly')
+                   intervals='monthly')
 
 
 magplasma_l2.__doc__ = _docstring(
     'STA_L2_MAGPLASMA_1M',
     'STEREO IMPACT/MAG Magnetic Field and PLASTIC Solar Wind Plasma Data.')
+
+
+def let_l1(spacecraft, starttime, endtime):
+    return _stereo(starttime, endtime, spacecraft, 'L1_LET')
+
+
+let_l1.__doc__ = _docstring('STA_L1_LET',
+                            """STEREO IMPACT/LET Level 1 Data
+
+    Note that the energies are given in units MeV/n and the 
+    intensities in units (1/(cm^2 s sr MeV/nuc)). The astropy
+    units cannot consistently give a unit per nucleon""")
+
+
+def sept_l1(spacecraft, starttime, endtime):
+    units = OrderedDict([('Heater_NS', u.deg_C),
+                         ('Heater_E', u.deg_C)])
+    sept = _stereo(starttime, endtime, spacecraft, 'L1_SEPT', units=units)
+    data = sept.to_dataframe()
+    data['Epoch_NS'] = util.epoch_to_datetime(data['Epoch_NS'].values)
+    data['Epoch_E'] = util.epoch_to_datetime(data['Epoch_E'].values)
+
+    return util.units_attach(data, sept.units)
```

Comment:
As I wrote earlier, I think that it would be best to have both of the Epochs separately in addition to the index field, as there is no guarantee (as far as I know) that the Epoch_NS and Epoch_E are always the same. The modified dataframe is now joined with the units in same way as in the original object.